src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        return userService.findByEmail(email) != null
+                && userService.findByEmail(email).getPassword().equals(password);
```

Comment:
```suggestion
        User user = userService.findByEmail(email);
        return user != null
                && user.getPassword().equals(password);
```

don't call method twice, you can create local variable